model/rbac.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package model
+
+const (
+	RBACHeaderDeploymentsGroups = "X-MEN-RBAC-Deployments-Groups"
+	RBACHeaderInvetoryGroups    = "X-MEN-RBAC-Inventory-Groups"
```

Comment:
could we have in the comments here ahy we are using two sets of groups from the headers?